slave/build_images.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/env python
+
+import sys
+import time
+from uuid import UUID
+from datetime import datetime
+import yaml
+from common import (
+    wait_for_image, rackspace_is_ready, load_manifest
+)
+
+from libcloud.compute.base import NodeImage
+from libcloud.compute.deployment import ScriptDeployment
+
+# Watch out for dodgy providers that mangle your tag names (I'm looking *right*
+# at you Rackspace).
+BASE_NAME_TAG = 'base_name'
```

Comment:
https://github.com/ClusterHQ/build.clusterhq.com/pull/81 makes that adjustment.  We also manually added `base_name` tags to all of the production images that are already on EC2.